### PR TITLE
Move unreachable warning before return

### PR DESCRIPTION
**Pull request by joren485**
`src/sentry/web/frontend/accounts.py` contains the following code:
``` 
    if request.method == 'POST' and ratelimiter.is_limited(
        u'accounts:recover:{}'.format(extra['ip_address']),
        limit=5,
        window=60,  # 5 per minute should be enough for anyone
    ):
        return HttpResponse(
            'You have made too many password recovery attempts. Please try again later.',
            content_type='text/plain',
            status=429,
        )
        logger.warning('recover.rate-limited', extra=extra)
```
`logger.warning('recover.rate-limited', extra=extra)` is called after the `return`, and thus never reached. This PR moves the line above the return.

### CODE REVIEW

**Review by markstory — APPROVED** [review]
Just waiting on CI before merging this.

**Comment by joren485** [comment]
It looks like travis failed on a `apt-get` command. @markstory could you restart the build?

**Pull request merged**
